sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1513,7 +1513,7 @@ object SQLConf {
         "1. pyspark.sql.DataFrame.toPandas " +
         "2. pyspark.sql.SparkSession.createDataFrame when its input is a Pandas DataFrame " +
         "The following data types are unsupported: " +
-        "BinaryType, MapType, ArrayType of TimestampType, and nested StructType.")
+        "BinaryType, ArrayType of TimestampType, and nested StructType.")
```

Comment:
It seems we already support BinaryType? I saw @BryanCutler updating arrow to v0.15.1.

Reply:
Fixed